.travis.yml

```
@@ -2,8 +2,6 @@ language: python
 sudo: false
```

```suggestion
sudo: false
```

This is deprecated stuff in TravisCI without purpose any more.

---

Oh sorry, I meant to remove this line entirely, but instead I removed one space in the github commit suggestion